docs/Integrations/Authentication/Concepts/authentication-approaches.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Authentication Approaches
+sidebar_position: 1
+
+---
+
+# Authentication Approaches
```

Comment:
Would be nice to have something visual on this page, maybe even a table with pros/cons for each approach since it's a lot of text. But that can be lower priority

Reply:
I've added images to explain how the method works, but may add more content & graphics in a later PR, not focusing on it too much at the moment.